proto/kiichain/rewards/v1beta1/params.proto

```
@@ -2,12 +2,35 @@ syntax = "proto3";
 
 // Params defines the parameters for the rewards module.
 message Params {
-  // Denom used
+  // Denom used for emissions
   string token_denom = 1;
-}
\ No newline at end of file
+
+  // Target fraction of supply staked (e.g. "0.67")
+  string goal_bonded = 2 [
+    (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
```

Buf (1.72.0)

[error] cannot find `gogoproto.customtype` in this scope

(COMPILE)